cmd/server-mux.go

```diff
@@ -457,6 +460,8 @@ func (m *ServerMux) ListenAndServe(certFile, keyFile string) (err error) {
 			// Execute registered handlers, update currentReqs to keep
 			// tracks of current requests currently processed by the server
 			atomic.AddInt32(&m.currentReqs, 1)
+			lmsg := fmt.Sprintf("serving for %s", r.URL.String())
+			log.logger.Info(lmsg)
```

Comment:
Right, as described in the pull request I did not want to change the default behavior. My next change was to make the log level configured via JSON though comments from harshavardhana make it sound like my changes are unwanted to begin with.